pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -1793,8 +1798,9 @@ def test_dti_add_offset_tzaware(self, tz_aware_fixture, box_with_datetime):
         expected = DatetimeIndex(['2010-11-01 05:00', '2010-11-01 06:00',
                                   '2010-11-01 07:00'], freq='H', tz=timezone)
 
-        dates = tm.box_expected(dates, box_with_datetime)
-        expected = tm.box_expected(expected, box_with_datetime)
+        # FIXME: these raise ValueError with transpose=True
```

Comment:
an you assert this as well? these are legit errors where you an't broadcast right? maybe have a nice error message then?

Reply:
The problem here is a bug in DataFrame (at least one Issue about it):

```
dti = pd.date_range('2016-01-01', periods=3, tz='US/Pacific')
df = pd.DataFrame(dti)
df.T
>>> df.T
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/site-packages/pandas/core/frame.py", line 2467, in transpose
    return super(DataFrame, self).transpose(1, 0, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/pandas/core/generic.py", line 686, in transpose
    new_values = self.values.transpose(axes_numbers)
  File "/usr/local/lib/python2.7/site-packages/pandas/core/base.py", line 708, in transpose
    nv.validate_transpose(args, kwargs)
  File "/usr/local/lib/python2.7/site-packages/pandas/compat/numpy/function.py", line 54, in __call__
    self.defaults)
  File "/usr/local/lib/python2.7/site-packages/pandas/util/_validators.py", line 218, in validate_args_and_kwargs
    validate_kwargs(fname, kwargs, compat_args)
  File "/usr/local/lib/python2.7/site-packages/pandas/util/_validators.py", line 157, in validate_kwargs
    _check_for_default_values(fname, kwds, compat_args)
  File "/usr/local/lib/python2.7/site-packages/pandas/util/_validators.py", line 69, in _check_for_default_values
    format(fname=fname, arg=key)))
ValueError: the 'axes' parameter is not supported in the pandas implementation of transpose()
```

Following @gfyoung's suggestion above, I am making a new fixture `box_transpose_fail` to parametrize over both box and transpose, xfailing the `(DataFrame, True)` pair.

Reply:
ok, so its the 1-D ones which are failing. I believe there is an open issue about transpose and tz DTI (in a frame) outstanding.